domains/transfer/Transfer.API/API.IntegrationTests/DbContextExtension.cs

```diff
@@ -33,10 +33,13 @@ public static async Task<List<T>> RepeatedlyQueryUntilCountIsMet<T>(this DbConte
 
     public static async Task TruncateTransferAgreementsTables(this ApplicationDbContext dbContext)
     {
-        var agreementsTable = dbContext.Model.FindEntityType(typeof(TransferAgreement))!.GetTableName();
-        var historyTable = dbContext.Model.FindEntityType(typeof(TransferAgreementHistoryEntry))!.GetTableName();
+        var transfersTable = dbContext.Model.FindEntityType(typeof(TransferAgreement))!.GetTableName();
+        var historyEntriesTable = dbContext.Model.FindEntityType(typeof(TransferAgreementHistoryEntry))!.GetTableName();
 
-        await dbContext.Database.ExecuteSqlRawAsync($"TRUNCATE TABLE \"{agreementsTable}\" CASCADE");
-        await dbContext.Database.ExecuteSqlRawAsync($"TRUNCATE TABLE \"{historyTable}\"");
+        var transfersSql = $"TRUNCATE TABLE \"{transfersTable}\" CASCADE";
+        var historyEntriesSql = $"TRUNCATE TABLE \"{historyEntriesTable}\"";
+
+        await dbContext.Database.ExecuteSqlRawAsync(transfersSql);
+        await dbContext.Database.ExecuteSqlRawAsync(historyEntriesSql);
     }
```

Comment:
Why this change?

Reply:
After upgrading from ef-core 7 to ef-core 8, the linter is complaining about how we execute raw sql statements, asking us to make it safe, from SQL injection. I refactored it, so it isn't complaining anymore, and so it is safe.